ui/components/prsidebar/prsidebar.go

```diff
@@ -98,7 +97,9 @@ func (m *Model) renderChecksPill() string {
 	} else if status == "PENDING" {
 		return pillStyle.Copy().
 			Background(styles.DefaultTheme.FaintText).
-			Render(constants.WaitingGlyph + " Checks")
+			Foreground(styles.DefaultTheme.MainText).
+			Faint(true).
+			Render(" Checks")
```

Comment:
Can you set the waiting glyph in the constants pkg? Instead of hardcoding it here

Reply:
The reason I hard coded it here was to fix it not displaying properly as I described in the PR description.  Before, it looked like this:

![image](https://user-images.githubusercontent.com/25914066/170786017-5a598954-58f3-47f9-a93a-715ee6c10eae.png)

But hard coding it makes it display correctly:

![image](https://user-images.githubusercontent.com/25914066/170785607-d9a941c5-d938-4ef8-9150-046a9d27ef4d.png)

Reply:
If you want, I could make constants for each of the icons used where each icon is just the actually glyph.  Then it could be referenced everywhere since it is just a string and not a lipgloss style.

Reply:
Ah ok, yeah the constants are with the styles already. Cool...
You could make the glyphs a constant but maybe that's just overkill...